test/mako/stub-sidecar/main.go

```diff
@@ -19,104 +19,151 @@ package main
 import (
 	"context"
 	"encoding/csv"
+	"flag"
 	"fmt"
-	"log"
-	"net"
-	"os"
+	"net/http"
 	"strings"
-	"sync"
 
 	"github.com/golang/protobuf/jsonpb"
+	mako "github.com/google/mako/spec/proto/mako_go_proto"
+
+	"log"
+	"net"
+	"sync"
 
 	"google.golang.org/grpc"
+
+	"knative.dev/pkg/test/mako/config"
 	qspb "knative.dev/pkg/third_party/mako/proto/quickstore_go_proto"
```

Comment:
It looks `quickstore_go_proto` is vendored, can we change it to `github.com/google/mako/proto/quickstore/quickstore_go_proto` and remove the `third_party`?

Reply:
quickstore_go_proto doesn't contain all stuff needed

Reply:
There are two quickstore_go_proto packages that are different:

https://github.com/knative/pkg/blob/master/vendor/github.com/google/mako/internal/quickstore_microservice/proto/quickstore_go_proto/quickstore.pb.go
https://github.com/knative/pkg/blob/master/vendor/github.com/google/mako/proto/quickstore/quickstore_go_proto/quickstore.pb.go

The one in third_party is a copy of the first one with `internal` in the path. Go won't allow code in a different path to import internal packages, so we need to copy it to third_party.

Reply:
So I can leave as it is now?

Reply:
Sure, let's keep it.